CONTRIBUTING.md

```diff
@@ -32,6 +32,19 @@ Your pull request should:
 * Follow the code conventions descriped in [Coding guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines)
 * To avoid line ending issues, set `autocrlf = input` and `whitespace = cr-at-eol` in your git configuration
 
+## Contributing `lib.d.ts` fixes
+ 
+The library sources are in: [src\lib\](https://github.com/Microsoft/TypeScript/tree/master/src/lib)
+
+To build the library files, run
+```Shell
+jake lib
+```
+
+`src\lib\dom.generated.d.ts` and `src\lib\webworker.generated.d.ts`:

```

Comment:
Looks like this should be a three-hash heading. Maybe that breaks some linter rule?